transforms/code/header_cleanser/README.md

```diff
@@ -0,0 +1,11 @@
+# Legal Removal Transform 
+The Legal Removal transforms 
+Detect and remove license and copyright of input data.
+Per the set of 
+[transform project conventions](../../README.md#transform-project-conventions)
+the following runtimes are available:
+
+* [python](python/README.md) - provides the base python-based transformation 
+implementation.
+* [ray](ray/README.md) - enables the running of the base python transformation
+in a Ray runtime
```

Comment:
Can you elborate on the implementation, 

* What is the dependent library,
* what is the api used for detecting the header 
*  what is the processing applied on that output of api by this module
* add an example for module execution through command-line
* sample input
* sample output